src/auto_scheduler/loop_state.cc

```diff
@@ -210,6 +387,16 @@ Array<Iterator> State::DoSplitStepCommon(int stage_id, int iter_id,
                      Stage(stage->op, stage->op_type, new_iters, stage->compute_at, stage->attrs));
   pstate->concrete &= concrete;
 
+  // We have to update the iterator relations in attach map, these two vectors keep the replacement
+  // mapping
```

Comment:
```suggestion
  // Use two vectors to represent the iterator relation before and after the split
  // in order to update the attach_map
```

We may also need to mention in the split function defintion in `loop_state.h` that if the attached target iterator is split, then the innermost split iterator would be the new attached target.